src/idl_gen_cpp.cpp

```diff
@@ -591,6 +591,20 @@ class CppGenerator : public BaseGenerator {
                       "std::string";  // Only for custom string types.
   }
 
+  bool NativeVectorDefined(const FieldDef *field) {
+    auto attr = field ? field->attributes.Lookup("cpp_vec_type") : nullptr;
```

Comment:
When can field be null? maybe better such that it is never null?

Reply:
In theory it's never null, I just took the check over from the other similar functions.
I don't think it's a problem to have it there, so I've kept it once in FieldStringLookup(), just to be safe.